README.md

```diff
@@ -15,83 +15,198 @@ npm run test
+To gather average FPS time, we just use the `FPS` function from `TraceModel`.
+For FPS we don't need to specify any components to look for as it will just grab the FPS for the entire page.
 
-		const browser = await puppeteer.launch({headless: true});
-		const page = await browser.newPage();
-		await page.setViewport({
-			width: 1920,
-			height: 1080
-		});
+### FID
 
-		await page.goto('http://localhost:8080/picker');
-		await page.tracing.start({path: filename, screenshots: false});
-		//interactions
-		//...
+First Input Delay (FID) measures the time from when a user first interacts with a page (i.e. when they click a link, tap on a button, or use a custom, JavaScript-powered control) to the time when the browser is actually able to begin processing event handlers in response to that interaction.
+To get FID we use the `FID` function from `TraceModel`.
 
-		await page.tracing.stop();
-		await browser.close();
+### CLS
 
-		// FPS
-		const actualFPS = FPS(filename);
-		TestResults.addResult({component: 'Picker', type: 'Frames Per Second', actualValue: actualFPS});
+Cumulative Layout Shift (CLS) is a measure of the largest burst of layout shift scores for every unexpected layout shift that occurs during the entire lifespan of a page.
+A layout shift occurs any time a visible element changes its position from one rendered frame to the next.
+To get CLS we use the `CLS` function from `TraceModel`.
 
-		// Update
-		const actualUpdateTime = Update(filename, 'Changeable');
-		TestResults.addResult({component: 'Picker', type: 'Update', actualValue: actualUpdateTime});
+### Example
 
-	});
-```
+Each component is tested repeatedly for both `click` and `keypress` events to measure average FPS. 
+FID and CLS are tested in the same test because they both typically require interactions. We can check the React Devtools to see which component is at the top of a specific component.
+DCL, FCP and LCP are also tested together as they are measured at page load time.
 
-### Mount
+Results can be found in `testResults` folder. 
 
-Mounting is very is pretty simple. We just need to find the top component for the component we're trying to test like `Mount(filename, 'Changeable');`. We can check the React Devtools to see which component is at the top of a specific component.
+### Comparing results 
+
+Test results are compared to the optimum values which are stored in global variables declared in `puppeteer.setup` file.
+
+```Metrics threshholds:```
+- global.maxCLS = 0.1; 
+- global.maxDCL = 2000; 
+- global.maxFCP = 1800; 
+- global.maxFID = 100; 
+- global.minFPS = 20; 
+- global.maxLCP = 2500;
 
 ```javascript
-	it('should mount picker under threshold', async () => {
-		const filename = getFileName('Picker');
-
-		const browser = await puppeteer.launch({headless: true});
-		const page = await browser.newPage();
-		await page.setViewport({
-			width: 1920,
-			height: 1080
+const TestResults = require('../TestResults');

```

Comment:
I think this part is "### Example"
Is it intended as "Comparing results"?

Reply:
Comparing results was supposed to be part of the Example section. I formatted the title wrongly, and now I think we can omit it altogether. Thanks!